src/login/pam_systemd.c

```diff
@@ -211,26 +218,45 @@ static int export_legacy_dbus_address(
         return r;
 }
 
+static int append_session_cg_limit(sd_bus_message *m, const char *limit, const char *name) {
+        uint64_t val;
+        int r;
+
+        if (!isempty(limit)) {
+                r = cg_weight_parse(limit, &val);
```

Comment:
i am pretty sure we should log here if we can't parse this stuff, with the pam logging functions

Reply:
The logging is done after the function is called, according to the return code. I did it like that so I can log which PAM object it failed to parse. Otherwise, I would have to pass it to the function. I don't really care, either works fine for me.